14_functions_lab/03.py

```diff
@@ -1,5 +1,14 @@
-"""
-Write a function that calculates the sum
-of the 10th digit from all arguments passed to it
-"""
-
+def TensSum(*list):
+    s = 0
+    for number in list:
+        if type(number) == int:

```

Comment:
Leaving it numeric yields a simpler solution: `(number / 10) % 10` is exactly the digit you need